src/Module6/CustomConfiguration/Enum/ProviderType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace CustomConfiguration.Enum
+{
+    public enum ProviderType
+    {
+        File = 0,
+        Xml = 1,
+        Json = 2
+    }
```

Comment:
According to task, you should support AppConfig reading/writing , can't find such functionality in your code